internal/db/schema/migrations/oss/postgres/52/01_worker_operational_state.up.sql

```diff
@@ -0,0 +1,76 @@
+begin;
+
+create table server_worker_operational_state_enm (
+  state text primary key
+  constraint only_predefined_operational_states_allowed
+    check (
+      state in (
+        'active',
+        'shutdown',
+        'unknown'
+      )
+    )
+);
+comment on table server_worker_operational_state_enm is
+  'server_worker_operational_state_enm is an enumeration table for worker operational states.';
+
+insert into server_worker_operational_state_enm (state) values
+  ('active'),
+  ('shutdown'),
+  ('unknown');
+
+alter table server_worker
+  add column operational_state text not null default 'active'
+    constraint server_worker_operational_state_enm_fkey
+      references server_worker_operational_state_enm (state)
+      on delete restrict
+      on update cascade;
+
+drop view server_worker_aggregate;
+-- Updates view created in 51/01_server_worker_release_version.up.sql to add the worker operational state
```

Comment:
Can you add a comment in `51/01_server_worker_release_version.up.sql` that the has been updated?